.pre-commit-config.yaml

```diff
@@ -30,11 +30,11 @@ repos:
         files: ^pandas
         exclude: ^pandas/tests
         args: [--select, "ANN001,ANN2", --fix-only, --exit-non-zero-on-fix]
+    -   id: ruff
+        args: [--exit-non-zero-on-fix, --preview, "--ignore=E,F841,RUF"]
```

Comment:
Can this just select pylint instead?

Reply:
it will override both the select and ignore section in `pyproject.toml` and select all pylint rules instead. I can't figure out a way to select only pylint rules while still respect the ignore section. 

How about adding a comment saying that only pylint rules need to be enabled in this hook and the rest can be ignored? We'll just need to update the ignore list when `ruff` is updated and something fails.

Reply:
@mroeschke I tried the new config override feature in `ruff` with `[--exit-non-zero-on-fix, --preview, --config, "lint.select=['PL']"]` but it still doesn't work for our case because both the select and ignore section in `pyproject.toml` will be resolved first, then they will be overriden together by `--config, "lint.select=['PL']"` in the command line, which result in a similar outcome to passing `--select=PL` directly to the command line. So I keep the old method to keep the pipeline passed.

Let me know what I should do next to get this PR merged.